From d8e4f575e4967a2303812529a866b89ad2fe26f5 Mon Sep 17 00:00:00 2001
From: Xaver Hugl <xaver.hugl@gmail.com>
Date: Tue, 7 Mar 2023 23:43:21 +0100
Subject: [PATCH] widgets/renamefiledialog: set number limit again

Not setting a maximum makes it default to 99, which is not what we want.
Instead, set it to a billion, which should be enough for everyone

BUG: 466636


(cherry picked from commit 04e31155574939dc0be2078b5f17c36791ef57e5)
---
 src/widgets/renamefiledialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/widgets/renamefiledialog.cpp b/src/widgets/renamefiledialog.cpp
index ad6eafbaa0..ceb0748755 100644
--- a/src/widgets/renamefiledialog.cpp
+++ b/src/widgets/renamefiledialog.cpp
@@ -132,6 +132,7 @@ RenameFileDialog::RenameFileDialog(const KFileItemList &items, QWidget *parent)
         mainLayout->addWidget(infoLabel);
         d->spinBox = new QSpinBox(page);
         d->spinBox->setMinimum(0);
+        d->spinBox->setMaximum(1'000'000'000);
         d->spinBox->setSingleStep(1);
         d->spinBox->setValue(1);
         d->spinBox->setDisplayIntegerBase(10);
-- 
GitLab

